rewards/choices.go

```diff
@@ -0,0 +1,83 @@
+package rewards
+
+import (
+	"context"
+
+	appctx "github.com/brave-intl/bat-go/utils/context"
+	"github.com/brave-intl/bat-go/utils/logging"
+	"github.com/shopspring/decimal"
+)
+
+var (
+	choiceTable = [][]float64{
```

Comment:
Can you provide a comment explaining what this is?

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/407/commits/2db23eb4cc994da49259718432fffc43f2808d41